Profiling/profile-dir.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
I do not think this is necessary. It adds some complexity to the repo and I do not clearly see the benefits. You can use the arguments of the rebench command to select which benchs, of which VMs and what number of iterations to run